app/controllers/application_controller.rb

```diff
@@ -28,4 +29,10 @@ def authenticate_admin!
   def user_signed_in?
     current_user.present?
   end
+
+  def title
+    return "#{@title} - #{t('layouts.application.title_suffix')}" if @title.present?
+
+    t('layouts.application.title')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.